src/main/java/net/pms/formats/v2/AudioProperties.java

```diff
@@ -83,6 +116,33 @@ public void setNumberOfChannels(String mediaInfoValue) {
 	}
 
 	/**
+	 * Get bits per sample for this audio track.
+	 * @return bits per sample (default is 16)
+	 */
+	public int getBitsperSample() {
+		return bitsperSample;
+	}
+
+	/**
+	 * Set bits per sample for this audio track.
+	 * @param bitsperSample bits per sample to set
+	 */
+	public void setBitsperSample(int bitsperSample) {
+		if (bitsperSample < 1) {
+			throw new IllegalArgumentException("Bits per sample can't be less than 1.");
+		}
```

Comment:
@Sami32 I'm not sure that this is a good idea. 0 is generally used as "unknown". This will probably cause some exceptions. 

I actually don't like setting the defaults like it was already done here, it's better to let the default be 0 and then return the default (e.g 16) in the getter if the value is actually 0. In that way, it's possible to tell the difference from a default/unknown value and the actual known value 16.